release-notes/Cloud/DCP_change_log.md

```diff
@@ -6,6 +6,83 @@ uid: DCP_change_log
 
 The DataMiner Cloud Platform gets updated continuously. This change log can help you trace when specific features and changes have become available.
 
+#### 27 September 2022 – New feature – DataMiner Teams bot support for custom commands [ID_34518]
+
+If CoreGateway version 2.11.0 or higher and FieldControl version 2.8.1 or higher are installed (included in Cloud Pack version 2.8.0), the DataMiner Teams bot now allows you to display and run custom commands with dynamic user input configured in your cloud-connected DataMiner System.
+
+You can do so using the Teams bot commands *show command \<command name\>*, *show command*, and *run command \<command name\>*.
+
+To add a command to your DMS, create an Automation script in the folder "bot" in the DMS. For examples of such scripts, refer to [ChatOps Extensions](https://github.com/SkylineCommunications/ChatOps-Extensions) on GitHub.
```

Comment:
can refer directly to CustomCommandExamples https://github.com/SkylineCommunications/ChatOps-Extensions/tree/main/CustomCommandExamples

Reply:
same for the other file

Reply:
Any further changes needed after my latest commit?